install/0000_90_cluster-version-operator_02_servicemonitor.yaml

```diff
@@ -106,4 +112,5 @@ spec:
         changes(cluster_operator_up{job="cluster-version-operator"}[2m]) > 2
       for: 10m
       labels:
+        namespace: openshift-cluster-version
```

Comment:
`ClusterOperatorFlapping` already has `namespace` in Telemetry (this time via `cluster_operator_up`), so I'd rather drop this line from the commit as redundant.

Reply:
Thank you, fixed in https://github.com/openshift/cluster-version-operator/pull/800/commits/d525eb97a2c971b4837210963d86d3b617452891.